### ISSUE TITLE
Handle leak inside psutil_get_process_data

### ISSUE THREAD

**Issue by xBeAsTx** [giampaolo/psutil#870]
https://github.com/giampaolo/psutil/blob/master/psutil/arch/windows/process_info.c#L436
Here is open process:
```
hProcess = psutil_handle_from_pid(pid);
```

and if all actions was succesful process handle not closing https://github.com/giampaolo/psutil/blob/master/psutil/arch/windows/process_info.c#L668
```
    *pdata = buffer;
    *psize = size;

    return 0;

error:
    if (hProcess != NULL)
        CloseHandle(hProcess);
    if (buffer != NULL)
        free(buffer);
    return -1;
}
```

Only if errors occured.

**Comment by giampaolo** [comment]
Good catch. Up for making a PR? =)

**Comment by giampaolo** [comment]
Fixed in #871.
